routes/event_details.js

```diff
@@ -1,19 +1,22 @@
-const request = require('request');
+const axios = require('axios');
 
-module.exports = (req, res) => {
-  const url = `https://nazareth-open-tourism-platform.herokuapp.com/api/v1/events/${req.params.id}`;
-
-  request.get(url, (error, result) => {
-    if (error) {
-      res.send(error);
-    } else {
-      res.render('event_details', {
-        title: 'Event Details',
-        eventText: JSON.parse(result.body)[req.params.lang],
-        event: JSON.parse(result.body),
-        place: JSON.parse(result.body).place[req.params.lang],
-        back: req.headers.referer
-      });
-    }
-  });
+module.exports = async (req, res) => {
```

Comment:
Keep PRs focused. If the whole code-base is moving to async await then make a PR dedicated to just that, rather than pushing it bit by bit with other PRs. :+1: